lstchain/scripts/longterm_dl1_check.py

```diff
@@ -944,6 +944,12 @@ def plot(filename='longterm_dl1_check.h5', batch=False, tel_id=1):
                                )
     r30_lowlim = reference_rate_above_30 * (1 - rate_above_30_tolerance)
     r30_upplim = reference_rate_above_30 * (1 + rate_above_30_tolerance)
+    
+    # The outermost pixels, on the camera corners, just because of their position, have 
+    # lower values of this rate of >30 pe pulses. We just set a looser lower limit 
+    # requirement (we just halve the value) on pixels with id >= 1800, to avoid lots of 
+    # unnecessary warnings:
+    r30_lowlim[:, 1800:] = 0.5 * r30_lowlim[:, 1800:]
```

Comment:
The camera geometry has a mask for the camera boundary pixels:

```
cam.get_border_pixel_mask(width=1)
```

Reply:
> The camera geometry has a mask for the camera boundary pixels:
> 
> ```
> cam.get_border_pixel_mask(width=1)
> ```

It is not exactly the edge pixels (regardless of the width value), but those close to the 6 corners, that have lower values. I now changed the number of affected pixels:

![image](https://user-images.githubusercontent.com/14164282/147993025-b53ac282-0f87-47cc-9ce9-8d7eb641729c.png)

This is just to avoid many useless warnings (and it is not worth to make now a more complicated parametrization of the expected value for just those few pixels), or a distance-dependent mask (because the cut has to be displayed on the "value vs. pixel index" plots)